pom.xml

```diff
@@ -153,6 +153,8 @@
     <httpcomponents.client.version>4.5.13</httpcomponents.client.version>
     <httpcomponents.core.version>4.4.13</httpcomponents.core.version>
     <immutables.version>2.9.2</immutables.version>
+    <httpcomponents5.core.version>5.3</httpcomponents5.core.version>
```

Comment:
do we now have a mix of httpcomponents5 and httpcomponents  v4?

Reply:
I don't know why dependencies on httpcomponents5 were added in the original PR, because after removing those from pom.xmls all the code was able to compile without changes :)